server/test/index.test.js

```diff
@@ -3,6 +3,9 @@ import chai, { expect } from 'chai';
 import chaiHttp from 'chai-http';
 import app from '../src/index';
 
+const NoteId = '7139d3af-b8b4-44f6-a49f-9305791700f4';
+const wrongNoteId = '7139d3af-b8b4-44f6';
+const applicationUrl = '/api/v1/notes/';
```

Comment:
While I understand that this aims at reducing redundancy, please can write this in the test and not here. We can also agree to do this as a team. Thanks